### DISCUSSION TITLE
河南郑州来报道了，抱团取暖，太难受了，银行现在开始催欠款了

### DISCUSSION THREAD

**Discussion by xclidongbo** [WeNeedHome/SummaryOfLoanSuspension#1096] [General]
河南的一起呀

**Comment by TALK1949** [comment]
现在啥情况，没什么动静了

**Reply by xclidongbo** [reply]
啥没动静， 停贷？国内新闻都屏蔽。

**Reply by NecZhang** [reply]
集体停贷了，官方不是说暂缓还贷了嘛，实际情况是咋样

**Comment by kajsnlk** [comment]
没有去起诉银行违规放贷吗？

**Comment by yybawang** [comment]
都没新闻了，封死了。所谓的起诉资源也都是皇帝的一个口令 yes/**N**o

**Comment by yybawang** [comment]
看得到郑州的烂尾数量是最多的，经历了核酸小屋+烂尾+白衬衫，微信里郑州的好友说已经选择去其它城市了

**Comment by 505Games-oss** [comment]
折腾呗，我看还能折腾出啥花样来

**Comment by shadowlgd** [comment]
金科中原的，停贷了，但是还没下文，郑州政府不办人事，都应该拉出去坐牢

**Reply by liusshuai** [reply]
哥，怎么样了，银行催款了吗

**Comment by neon9lyph** [comment]
预售房未交房之前，预售款不是存放在监管账户里的吗，现在全国私自挪用监管账户资金的行为，是被默许了吗

**Comment by ColdHimmel** [comment]
what the hell is going on in china

**Reply by daimou123** [reply]
The internal economy cannot be regulated

**Comment by xiaohao587** [comment]
贷款已经停了，爱咋咋滴，逼急了大不了一起死

**Reply by liusshuai** [reply]
我也想停，但我怕我之后工资都取不出来，公积金也直接被划走了

**Reply by Cegazhos** [reply]
> 我也想停，但我怕我之后工资都取不出来，公积金也直接被划走了

先更改还款账户嘛。

**Reply by liusshuai** [reply]
> > 我也想停，但我怕我之后工资都取不出来，公积金也直接被划走了
> 
> 先更改还款账户嘛。

我现在还款是一个银行，工资卡是另一个银行，停贷了会不会直接把我的另一个银行的钱划走呀

**Comment by xclidongbo** [comment]
已经第三个月了. 听之前欠6个月以上的人,说P事没有. 老赖也只是法院起诉后,才强制还款. 再说你楼都没盖起. 银行不可能拍卖你的房子.
